ANY API: Add testnets supported devrel operators (#911)
smartcontractkit/documentation: docs/Using Any API/array-response.md

   - The Chainlink Token address for _Rinkeby_ by calling the [`setChainlinkToken`](/docs/chainlink-framework/#setchainlinktoken) function.
   - The Oracle contract address for _Rinkeby_ by calling the [`setChainlinkOracle`](/docs/chainlink-framework/#setchainlinkoracle) function.

>    - The `jobId`: A specific job for the oracle node to run. In this case, the _id_ is a _string_ data type, so you must call a job that calls an API and returns a _string_. We are going to use a generic **_GET>string_** job that can be found [here](/docs/any-api-testnet-oracles/#jobs).

Reviewer: ```suggestion
   - The `jobId`: A specific job for the oracle node to run. In this case, the _id_ is a _string_ data type, so you must call a job that calls an API and returns a _string_. We will be using a generic **_GET>string_** job that can be found [here](/docs/any-api-testnet-oracles/#jobs).
```
